src/utils/context.js

```diff
@@ -22,6 +23,7 @@ export function getContext(program, subcommand, selectedDigitalObject) {
     doHome: DEFAULT_DO_HOME,
     processorHome: PROCESSOR_HOME,
     deploymentHome: DEFAULT_DEPLOY_HOME,
+    reconstructionHome: DEFAULT_RECONSTRUCT_HOME,
```

Comment:
I'm really not sure we need a dedicated directory for reconstruction. Even so, it should be designed to work in parallel.